core/tests/puppeteer-acceptance-tests/blog-post-draft-blogs.js

```diff
@@ -0,0 +1,32 @@
+const acceptanceTests = require("./utility-functions/puppeteer_utils.js");
+const testConstants = require("./utility-functions/testConstants.js");
+
+
+const blogEditOptions = "e2e-test-blog-post-edit-box";
+
+async function deleteDraftAsBlogAdmin() {
+  const user = await new acceptanceTests();
+  const page = await user.init();
```

Comment:
What do you need `page` for (in fact, what do you need this entire line for)?

Reply:
by this we are waiting for the browser page to asynchronously load, first I thought to call that init() function inside the constructor but since the js constructors are not asynchronous, so we need to explicitly call the init() function that is loading the browser instances asynchronously.